composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
@mibstar what @e0ipso suggested is a better idea. If you want since is a small code change,  added in the documentation.